<ReferenceInput>,< ReferenceArrayInput > ,<AutocompleteInput> validate required message not red as same as other input

**What you were expecting:**
when set validate={required()} in  ReferenceInput or ReferenceArrayInput or AutocompleteInput

```
   <ReferenceInput label="Service Provider" source="serviceProviderId" reference="serviceProviders" validate={required()}>
        <AutocompleteInput optionText="name" />
      </ReferenceInput>

   <ReferenceArrayInput
        label="Eligible Services"
        source="eligibleServiceIds"
        reference="services"
        validate={required()}
      >
        <AutocompleteArrayInput optionText="name" />
      </ReferenceArrayInput>

  <AutocompleteInput source="timeZone" choices={timeZoneChoices} validate={required()} />
```
<img width="233" alt="WeChat66eb1017b42fd008a19284b81795df4b" src="https://user-images.githubusercontent.com/19420622/67193903-b1c72b80-f428-11e9-9f58-691f1477e3f2.png">

the validation message display style not a  red color. At the same time ,ReferenceArrayInput and ReferenceInput do not have * after the label. AutocompleteInput have render the *
 
**Environment**

* React-admin version:3.0.0-beta.2

> Have you updated to the latest beta ?

just Now i update to last beta2.0 ,but failed ,following is the error.
```
./node_modules/ra-core/esm/util/renderWithRedux.js
Module not found: Can't resolve '@testing-library/react' in '/Users/liuzhichao/AnewWork/cowboy-wms/node_modules/ra-core/esm/util'
```
can you please tell me why ? thanks

--- a/packages/ra-ui-materialui/src/input/AutocompleteArrayInput.tsx
+++ b/packages/ra-ui-materialui/src/input/AutocompleteArrayInput.tsx
@@ -390,6 +390,7 @@ const AutocompleteArrayInput: FunctionComponent<
                                 },
                                 onFocus,
                             }}
+                            error={!!(touched && error)}
                             label={
                                 <FieldTitle
                                     {...labelProps}

--- a/packages/ra-ui-materialui/src/input/AutocompleteInput.tsx
+++ b/packages/ra-ui-materialui/src/input/AutocompleteInput.tsx
@@ -345,6 +345,7 @@ const AutocompleteInput: FunctionComponent<
                                 },
                                 onFocus,
                             }}
+                            error={!!(touched && error)}
                             label={
                                 <FieldTitle
                                     {...labelProps}

